frontend/src/components/SignUp/SignUpForm.jsx

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+import { withStyles } from '@material-ui/core/styles';
+import { ValidatorForm, TextValidator } from 'react-material-ui-form-validator';
+
+import 'typeface-roboto';
+
+const styles = theme => ({
+  form: {
+    marginTop: 50,
+  },
+  cardBody: {
+    padding: 20,
+  },
+  textField: {
+    marginLeft: theme.spacing.unit,
+    marginRight: theme.spacing.unit,
+    width: 300,
+  },
+  button: {
+    margin: theme.spacing.unit,
+  },
+});
+
+class SubmittedForm extends React.Component {
+  render() {
+    const {classes} = this.props;
+    return (
+      <div>
+        <Grid container justify='center' className={classes.form}>
+          <Typography variant="h4" align='center'>
+            {this.props.message}
+          </Typography>
+          <Grid container direction='column' alignContent='center'>
+            <Grid item>
+              <Button variant="outlined" href="/" className={classes.button}>
```

Comment:
Do not use <Button href="/"> to jump to another page. Our project is a SPA. Use the Link component from "react-router-dom".

Reply:
Why Button can't be used in this case? What are the benefits of using Link instead of href?

Reply:
When you use ..Button href="/".. , it reload a page... Try something like "..Button component={Link} to={'/'} "